### DISCUSSION TITLE
Connecting to Google Gemini

### DISCUSSION THREAD

**Discussion by michaelisvy** [spring-projects/spring-ai#1249] [General]
hello, 
thanks a lot for the great work on Spring AI! 
I have no issue to use Spring AI with OpenAI, but didn't manage to connect to Google Gemini. 

Using OpenAI, I have this configuration:

```
spring.ai.openai.api-key=${OPENAI_API_KEY}
spring.ai.openai.chat.options.model=gpt-4o
```

Using Gemini, I only found this config from the [Spring AI documentation](https://docs.spring.io/spring-ai/reference/api/chat/vertexai-gemini-chat.html):

spring.ai.vertex.ai.gemini.project-id=PROJECT_ID
spring.ai.vertex.ai.gemini.location=LOCATION
spring.ai.vertex.ai.gemini.chat.options.model=vertex-pro-vision
spring.ai.vertex.ai.gemini.chat.options.temperature=0.5

It doesn't tell me where to put the Google api key. What is the parameter for that?

Thanks!

**Comment by KotlinFactory** [comment]
See #1252
